backend/src/v5/models/modelSettings.js

```diff
@@ -109,6 +109,18 @@ Models.getModelType = async (ts, model) => {
 	return getModelType(item);
 };
 
+Models.getMembers = async (ts, model, excludeViewers) => {
+	const pipeline = [
```

Comment:
this aggregate is a bit overkill.

We know for sure that only one document will ever match here (since we're matching by id)

I think it's less operations if you just do a findOne and do a flatMap on the permissions array